packages/alice/lib/model/alice_configuration.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:alice/core/alice_logger.dart';
+import 'package:alice/core/alice_memory_storage.dart';
+import 'package:alice/core/alice_storage.dart';
+import 'package:flutter/widgets.dart';
+
+class AliceConfiguration {
```

Comment:
`AliceConfiguration` should probably mix in `EquatableMixin` in case someone wants to compare various configs?

Reply:
Fixed :)